### PR TITLE
Automatically reassign Motions to Vacate tasks back to motions attorneys when denied or withdrawn/dismissed

### PR DESCRIPTION
**Pull request by jcq**
Connects #12286
 
### Description
Updated `PostDecisionMotionUpdater` to automatically assign `DeniedMotionToVacateTask` or `DismissedMotionToVacateTask` to previously assigned motions attorney. In case prev motions attorney is inactive, they get assigned to `LitigationSupport` org.

Also updated frontend "Address Motion to Vacate" view to match design changes in #12286. 

### Acceptance Criteria
- [ ] Code compiles correctly

### CODE REVIEW

File: client/app/queue/QueueApp.jsx [hunk 1]
```diff
@@ -237,6 +238,19 @@ class QueueApp extends React.PureComponent {
 
   routedAssignToPulacCerullo = (props) => <AssignToView isTeamAssign assigneeAlreadySelected {...props.match.params} />;
 
+  routedReturnToLitSupport = (props) => {
+    return (
+      <ReturnToLitSupportModal
+        {...props.match.params}
+        onCancel={() => props.history.goBack()}
+        onSubmit={({ instructions }) => {
+          // we'll flesh this out in future PR
```
Comment (leikkisa) [inline client/app/queue/QueueApp.jsx]: ➕

File: client/app/queue/mtv/MTVJudgeDisposition.jsx [hunk 2]
```diff
@@ -74,37 +77,39 @@ export const MTVJudgeDisposition = ({
...
       !disposition ||
       !instructions ||
-      !attorneyId ||
       (disposition === 'granted' && !vacateType) ||
       (disposition !== 'granted' && !hyperlink)
-    ) {
-      return false;
-    }
-
-    return true;
```
Comment (leikkisa) [inline client/app/queue/mtv/MTVJudgeDisposition.jsx]: ➕ to this refactor

Comment (leikkisa) [inline client/app/queue/mtv/MTVJudgeDisposition.jsx]: ➕ to this refactor

File: spec/feature/queue/motion_to_vacate_spec.rb [hunk 3]
```diff
@@ -265,7 +266,7 @@
         disposition: "denied",
         hyperlink: hyperlink
       )
-      new_task = DeniedMotionToVacateTask.find_by(assigned_to: drafting_attorney)
+      new_task = DeniedMotionToVacateTask.find_by(assigned_to: motions_attorney)
```
Comment (leikkisa) [inline spec/feature/queue/motion_to_vacate_spec.rb]: 👍

File: spec/workflows/post_decision_motion_updater_spec.rb [hunk 4]
```diff
@@ -85,59 +94,61 @@
...
-          expect(AbstractMotionToVacateTask.count).to eq 0
-          expect(DeniedMotionToVacateTask.count).to eq 0
-        end
+        subject.process
+        expect(task.reload.status).to eq Constants.TASK_STATUSES.completed
+        abstract_task = AbstractMotionToVacateTask.find_by(parent: task.parent)
+        attorney_task = DeniedMotionToVacateTask.find_by(parent: abstract_task)
+        expect(attorney_task).to_not be nil
+        expect(attorney_task.parent).to eq abstract_task
+        expect(attorney_task.assigned_by).to eq task.assigned_to
```
Comment (leikkisa) [inline spec/workflows/post_decision_motion_updater_spec.rb]: Haha true

File: spec/workflows/post_decision_motion_updater_spec.rb [hunk 5]
```diff
@@ -85,59 +94,61 @@
...
     context "when disposition is dismissed" do
       let(:disposition) { "dismissed" }
-      let(:assigned_to_id) { create(:user).id }
 
       it "should create post decision motion dismissed attorney task" do
         subject.process
         expect(task.reload.status).to eq Constants.TASK_STATUSES.completed
-        abstract_task = AbstractMotionToVacateTask.find_by(parent: task)
-        attorney_task = DismissedMotionToVacateTask.find_by(assigned_to_id: assigned_to_id)
+        abstract_task = AbstractMotionToVacateTask.find_by(parent: task.parent)
```
Comment (leikkisa) [inline spec/workflows/post_decision_motion_updater_spec.rb]: These are good examples of where we'll want to add org parent tasks in the future.

**Comment by jcq** [comment]
Updated screenshots:

Screen Shot 2019-10-22 at 16 41 28

**Comment by jcq** [comment]
> This looks really good to me ⭐️! The text on the "missing draft" alert looks a little big to me compared to design, but I'll leave that to design. Do you mind adding a couple of screenshots to the ticket before you merge?

That was a good catch on the font size in the Alert. It turns out there are styles built in for larger text on alerts that don't have a title. Rutvi agreed it looked weird, so we added a title in order to make it better.

**Comment by jimruggiero** [comment]
Return to motions attorney link in banner message requires a separate ticket be created. @jimruggiero will create new tickets to track new frontend and backend work.

**Pull request merged**